### DISCUSSION TITLE
How do i setup the cli version windows 10 ?

### DISCUSSION THREAD

**Discussion by neilanthonystuckey** [Marekkon5/onetagger#212] [Q&A] [answered]
I went through the GUI setup and it works fine.  I clicked on the CLI button under auto tag.  It tells me to put info into config.json. I assume this is in the C:\Program Files (x86)\one tagger ? Or would it be C:\Users\username\AppData\Roaming\OneTagger\OneTagger\config ? Or should this file be called settings.json instead of config.json in windows?

I am essentially trying to call onetagger from cli to first auto tag from cli then auto rename from cli once a day for automation.

thanks

Neil

**Comment by Marekkon5** [comment]
Hello, the cli and gui versions are separate, you can put the config.json to any folder, and then specify path using the `--config` argument in the cli version. Optionally you can put both the `onetagger-cli.exe` and `config.json` into one folder and use `-c config.json`

**Reply by neilanthonystuckey** [reply, marked as answer]
I was going crazy trying to figure this out. I think I had it right from the start as far as config goes.  This was a copy paste issue with windows notepad . Once I copied the config info with notepad ++ it worked. I think windows default notepad was messing up some characters or formatting ? Anywho it works now. Thanks for the fast response. And nice app.
